### PR TITLE
Added types for mcping-js

### PR DESCRIPTION
**Pull request by Inrixia**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/mcping-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Type definitions for mcping-js 1.5
+// Project: https://github.com/Cryptkeeper/mcping-js
+// Definitions by: Inrix <https://github.com/Inrixia>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 4.0
```
Comment (peterblazejewicz) [inline types/mcping-js/index.d.ts:6]:
```suggestion
// TypeScript Version: 4.0
```
why you're limiting usage of this package? What's the purpose of this? (I cannot see TS 4.0 specific language feature being used. Thsi header restricts usage IMO

Comment (Inrixia) [reply types/mcping-js/index.d.ts:6]: My bad, copied the definition from another file which has limitations. All good to remove that line

Comment (peterblazejewicz) [reply types/mcping-js/index.d.ts:6]: So please accept suggestion and remove the requirements, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Inrixia thx!

**Pull request merged**